beacon-chain/core/altair/epoch_precompute_test.go

```diff
@@ -325,20 +345,20 @@ func TestProcessRewardsAndPenaltiesPrecompute_GenesisEpoch(t *testing.T) {
 func TestProcessRewardsAndPenaltiesPrecompute_BadState(t *testing.T) {
 	s, err := testState()
 	require.NoError(t, err)
-	validators, balance, err := InitializeEpochValidators(context.Background(), s)
+	validators, balance, err := InitializePrecomputeValidators(context.Background(), s)
 	require.NoError(t, err)
 	_, balance, err = ProcessEpochParticipation(context.Background(), s, balance, validators)
 	require.NoError(t, err)
 	_, err = ProcessRewardsAndPenaltiesPrecompute(s, balance, []*precompute.Validator{})
-	require.ErrorContains(t, "validator registries not the same length as state's validator registries", err)
+	require.ErrorContains(t, "validator registries not the same length as beaconState's validator registries", err)
```

Comment:
```suggestion
	require.ErrorContains(t, "validator registries not the same length as state's validator registries", err)
```

Refactoring side-effect.